FakeItEasy.PrepareRelease/Program.cs

```diff
@@ -169,12 +169,12 @@ private static bool CrossReferenceIssues(ICollection<Issue> issuesInMilestone, I
 
             Console.WriteLine("Prepare release anyhow? (y/N)");
             var response = Console.ReadLine().Trim();
-            if (string.Equals(response, "y", StringComparison.InvariantCultureIgnoreCase))
+            if (string.Equals(response, "y", StringComparison.Ordinal))
```

Comment:
But but what if I type "Y"?

Reply:
That was a mistake on my part. Should have been `OrdinalIgnoreCase`.
But you already merged, so it'll have to be fixed later

Reply:
Fixed it directly on master (I know, it's bad... we don't have branch protection rules on this repo, and I took advantage of it)

Reply:
Oh, I wasn't really worried about typing "Y". I'm sure I'll type "y" (more likely "n').